materials/section-1/README.md

```diff
@@ -36,6 +36,16 @@ clojure.core/+
      (reduce1 + (+ x y) more)))
 ```
 
+### Nightcode
```

Comment:
Is it worth also adding some alternatives? Setting up VSCode for Clojure seems pretty simple (as seen [here])(https://spin.atomicobject.com/2017/06/22/clojure-development-with-visual-studio-code/), or even using emacs (as referenced in our [favourite book](https://www.braveclojure.com/basic-emacs/)).

Perhaps it's a little bit too close on time to expect people to get familiar with emacs but VSCode seems to be a decent alternative. There were lots of headaches with Nightcode last time round so giving a quick and easy alternative might be sensible

Reply:
Not sure about this. I don't use VSCode so I'm not sure what how to set up VSCode for Clojure 🤔 In the example article, the author doesn't use parinfer or paredit, which personally I see as a must :/